src/main/scala/cromwell/engine/backend/BackendType.java

```diff
@@ -1,5 +1,16 @@
 package cromwell.engine.backend;
 
 public enum BackendType {
-    LOCAL, JES, SGE;
+    LOCAL {
+        @Override
+        public String displayName() {
+            return "Local";
+        }
+    },
+    JES,
+    SGE;
```

Comment:
On purpose, the display name for LOCAL needs some beautification but JES and SGE are natural beauties.